pkg/apis/vpc/v1alpha1/subnet_types.go

```diff
@@ -113,6 +125,14 @@ type SubnetDHCPConfig struct {
 	Mode DHCPConfigMode `json:"mode,omitempty"`
 }
 
+// VLANExtension describes VLAN extension configuration for the VPC Subnet.
+type VLANExtension struct {
```

Comment:
These struct is in the NSX API Subnet Status too, with the same fields?

Reply:
Yes, NSX API schema is not out yet, these are based on Jame's example on his Policy API doc.
{
    "status": "SUCCESS",
    "realization_entities": [
        {
            "status": "SUCCESS",
            "path": "/orgs/default/projects/default/vpcs/test/subnets/s2/status",
            "realized_entity_path": "/infra/realized-state/realized-entities?intent_path=/orgs/default/projects/default/vpcs/test/subnets/s2"
        }
    ],
    "results": [
        {
            "gateway_address": "20.20.20.1/24",
            "network_address": "20.20.20.0/24",
            "ip_address_type": "IPV4",
             "vlan_extension": {
                  "vpc_gateway_connection_enable": false,
                  "vlan_id": 10
              }
        }
    ]
}

Reply:
What you pasted is the response of Subnet creation? We put fields in a "results" entry in Status? Are result fields in the NSX VPC Status?

Reply:
it is the response of "get Subnet Status" API.

Reply:
Ok